docs/procedures/rigid_waist_shift.md

```diff
@@ -0,0 +1,77 @@
+# LHC Local Coupling Corrections with a Rigid Waist Shift
+
+!!! note ""
+    Please keep in mind the [general checks](general_checks.md) for measurements.
+
+??? info "The Procedure in Short"
+
+      This methods aims to find the MQSX correction settings that would minimize betatron coupling and its impact on beam size *at IP*.
+      The method breaks the symmetry of the optics in the IR and forces local coupling RDTs to leak thoughout the machine, which makes them measurable through the $|C^{-}|$.
+
+
+      After global corrections are done and trimmed in the machine, one applies a rigid waist shift in a given IR and scans the colinearity knob for the value that minimises the $|C^{-}|$.
+      These settings, when taking away the rigid waist shift, will minimize local coupling and its impact at IP.
+
+## Preliminary Setup
+
+- [ ] <details class="nodeco"><summary>Do Global Corrections</summary>
+      <p> This procedure needs global corrections to be trimmed in the machine first, so optics and *global* coupling should be taken care of beforehand.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>**Optional:** Scan the Colinearity Knob to Check Conditions</summary>
+      <p> If time allows, ideally we would scan the colinearity knob to ensure we see very small variations of the $|C^{-}|$.
+      If strong variations are noticed, then the expected conditions for the procedure are not met: either the phase advance between left and right MQSXs is off, or the $\sqrt{\beta_x \beta_y}$ is significantly wrong at these elements.
+      </p></details>
+
+## Procedure Per IP
+
+One should do this for earch of IP1, IP2, IP5 and IP8 which are the ones requiring local corrections.
```

Comment:
Maybe sentence could be rephrased. E.g. this procedure is applied at every experimental insertion (IP1, IP2, IP5, IP8) where local corrections need to be established or checked. Different IPs cannot/ should not be done in parallel.

Reply:
Resolved with [028c859](https://github.com/pylhc/pylhc.github.io/pull/71/commits/028c8596e86bf663e15301c7ca432ee1b6122333)